### PR TITLE
IRGen: Sort the target-features string

### PR DESCRIPTION
**Pull request by aschwaighofer**
Function merging will not identify two otherwise equivalent if the order of the target-features list differs.

rdar://33057710

### CODE REVIEW

**Comment by aschwaighofer** [comment]
@swift-ci Please test and merge

File: lib/IRGen/IRGenModule.cpp [hunk 1]
```diff
@@ -817,6 +817,8 @@ llvm::AttributeSet IRGenModule::constructInitialAttributes() {
   std::vector<std::string> &Features = ClangOpts.Features;
```
Comment (eeckstein) [inline lib/IRGen/IRGenModule.cpp]: Wouldn't you want to make a copy before sorting? (i.e. omitting the '&')

Comment (aschwaighofer) [inline lib/IRGen/IRGenModule.cpp]: Good point.

**Comment by aschwaighofer** [comment]
@swift-ci Please test

**Review by eeckstein — APPROVED** [review]
LGTM!

**Pull request merged**